GameEngine/GameState.cs

```diff
@@ -6,19 +6,22 @@ namespace GameEngine
 {
     public class GameState
     {
-        public GameBoard Board { get; private set; }
-        public Deck Deck { get; private set; }
-        public IPlayer Player { get; private set; }
+        public GameBoard Board { get; }
+        public Deck Deck { get; }
+        public IPlayer Player { get; }
+        public int SunSpaces { get; }
         public int SunCounter { get; private set; }
-        public int GameSizeMultiplier { get; }
-        
-        public GameState(IPlayer player, Deck deck, int gameSizeMultiplier)
+
+        public GameState(int multiplier, params IPlayer[] player)
+            : this(GameOptions.FromMultiplier(multiplier), player) { }
+
+        public GameState(GameOptions options, params IPlayer[] players)
```

Comment:
❓ Why does this allow multiple players when only the first is used?

Reply:
Looking ahead to future stories. Removed because YAGNI.